.drone.jsonnet

```diff
@@ -193,7 +193,8 @@ local Pipeline(name, steps=[], depends_on=[], with_docker=true, disable_clone=fa
 
 local generate = Step('generate', target='generate docs', depends_on=[setup_ci]);
 local check_dirty = Step('check-dirty', depends_on=[generate]);
-local build = Step('build', target='talosctl-all kernel initramfs installer imager talos _out/integration-test-linux-amd64', depends_on=[check_dirty], environment={ IMAGE_REGISTRY: local_registry, PUSH: true });
+local uki_certs = Step('uki-certs', depends_on=[check_dirty], environment={ PLATFORM: 'linux/amd64' });
```

Comment:
does it affect _dirtiness_? do we not gitignore it?

Reply:
so we add a step, but nothing depends on it? should we just add it as a target to `generate` ? or run it in parallel, and make `check_dirty` depend on it?

Reply:
so we add a step, but nothing depends on it? should we just add it as a target to `generate` ? or run it in parallel, and make `check_dirty` depend on it?